modes/magit-todos/evil-collection-magit-todos.el

```diff
@@ -48,11 +48,11 @@
 (defun evil-collection-magit-todos-setup ()
   "Set up `evil' bindings for `magit-todos'."
   ;; magit-todos binds jT which prevents evil users from stepping into the section
-  (evil-collection-define-key nil 'magit-todos-section-map
```

Comment:
This seems like it would work as intended but I don't currently have this package installed.